templates/components/pager.html.twig

```diff
@@ -39,7 +39,8 @@
     {% endif %}
 {% endif %}
 
-{% set href = href ~ "&start=%start%" ~ additional_params %}
+{% set href_separator = '?' in href ? '&' : '?' %}
+{% set href = href ~ href_separator ~ "start=%start%" ~ additional_params %}
 {% if is_tab is defined and is_tab == true %}
    {% set href = "javascript:reloadTab('start=%start%" ~ additional_params ~ "');" %}
```

Comment:
Unless I miss something, this `href` may not even be used in the `limit.html.twig` template because it will be overriden in the `{% if is_tab is defined and is_tab == true %}` block.

Reply:
No, that's correct. When the pager is in a tab, it works fine because it overwrites the href rather than blindly appending new query parameters without a separator.